Hakim Resume Page PT. 1 Final .html

```diff
@@ -0,0 +1,86 @@
+<!DOCTYPE html>
+<html> 
+  <head>
+    <title>Hakim's Resume</title>  
+  </head>
+    <body> 
+      <div>  
+        <header>
+            <h1>Name: Hakim Mackey</h1>
+        </header>
+            
+        <br>
+            <section>
+              <h2>OBJECTIVE</h2>
+                <p>Seeking an opportunity in an entry level position in the IT industry where I can cultivate my training, communication and leadership abilities to enhance the success of your organization and become a viable asset.</p>
+            </section>
+        
+        <br>
+            <section>  
+              <h2>WORK EXPERIENCE</h2>
+                <ul>
+                  <h3>Philadelphia Revenue Department</h3>
+                    <ul>  
+                      <li><p>Tax Assessor - (2014 - Present)</p></li>
+                      <li><p>Revenue Investigator - (2010 - 2014)</p></li>
+                    </ul> 
+                </ul> 
+            </section>
+
+            <section>   
+              <h2>EDUCATION</h2>
+                <ul>
+                  <h3><strong>Community College of Philadelphia</strong></h3>
+                    <ul>
+                      <li><p>Associates in Liberal Arts (2011)</p></li>
+                    </ul>
+                </ul>
+            </section>
+            
+            <section>
+              <h2>SKILLS</h2>
+                <ul>
+                    <h3>Intermediate/Advanced Computer Skills</h3>
+                    <ul>
+                        <li><p>Microsoft Office (Word, Excel, PowerPoint, Publisher)</p></li>
+                        <li><p>Adobe Photoshop</p></li>
+                        <li><p>Adobe Premier</p></li>
+                    </ul>
+                </ul>
+            </section>
+            
+            <section>
+              <h2>LINKS</h2>
+                <ul>
+                  <a href="https://www.linkedin.com/in/mikail-mackey-8215034/">LinkedIn Profile</a>
+                </ul>
+            </section> 
+            
+            <div>
+            <section>
+              <figure>
+                <img src="Demos/img/Carl Madison.jpg" alt="Profile Pic" style="width:200px;height:200px;">
```

Comment:
File path and file name are wrong, it should be `src="FB profile pic.jpg"` based off of what you submitted (-2)